src/Lachain.Core/RPC/HTTP/Web3/AccountServiceWeb3.cs

```diff
@@ -44,10 +45,10 @@ public string GetBalance(string address, string tag)
         }
 
         [JsonRpcMethod("eth_getTransactionCount")]
-        public ulong GetTransactionCount(string from, string blockId)
+        public HexBigInteger GetTransactionCount(string from, string blockId)
```

Comment:
We have our own format utils,  I don't think it is a good idea to use other project formats in parallel, Remove string or JObject instead and format it as required

Reply:
fixed the issues in the latest commit